sympy/control/core.py

```diff
@@ -0,0 +1,269 @@
+"""
+A symbolic control toolbox.
+"""
+
+import sympy
+import mpmath
+
+# pylint: disable=invalid-name
+
+s, t = sympy.symbols('s, t')
+
+def array_to_poly(a, s):
+    """
+    Convert an array into a polynomial in s
+
+    :param a: array
+    :param x: variable
+    """
+    n = len(a)
+    p = 0
+    for i, c in enumerate(a):
+        p += c*s**(n - i - 1)
+    return p
+
+
+def tf(num, den):
+    """
+    Create a transfer function
+
+    :param num: an array of numerator coefficients with order [n, n-1, .., ]
+    :param den: an array of denominator coefficients with order [n, n-1, .., ]
+    """
+    return array_to_poly(num, s) / array_to_poly(den, s)
+
+
+class StateSpace(sympy.Tuple):
+    """
+    State space class, derives from Tuple.
+    """
+
+    def __init__(self, A, B, C, D, dt):
+        """
+        :param A: A matrix
+        :param B: B matrix
+        :param C: C matrix
+        :param D: D matrix
+        :param dt: sampling time, 0 for continuous
+        """
+        self.A = sympy.Matrix(A)
+        self.B = sympy.Matrix(B)
+        self.C = sympy.Matrix(C)
+        self.D = sympy.Matrix(D)
+        self.dt = dt
+
+    def simplify(self):
+        """
+        Simplify the state-space
+        """
+        self.A.simplify()
+        self.B.simplify()
+        self.C.simplify()
+        self.D.simplify()
+
+
+def ss(A, B, C, D, dt):
+    """
+    Create a state space represenation.
+    """
+    return StateSpace(A, B, C, D, dt)
+
+
+def ss2tf(ss):
+    """
+    Transform a state-space to a transfer function matrix
+
+    :param ss: State-space
+    :return : Transfer function matrix
+    """
+    n = ss.A.shape[0]
+    I = sympy.eye(n)
+    G = (ss.C*(s*I - ss.A).inv()*ss.B + ss.D)
```

Comment:
There are much more efficient ways to calculate this. It is generally not good practice to use `inv()` like this, especially in a core alg.